pkg/sqlite/load.go

```diff
@@ -575,20 +575,20 @@ func (s *sqlLoader) addPackageChannels(tx *sql.Tx, manifest registry.PackageMani
 	var errs []error
 
 	if _, err := addPackage.Exec(manifest.PackageName); err != nil {
-		errs = append(errs, err)
+		errs = append(errs, fmt.Errorf("failed to add package %q: %s", manifest.PackageName, err.Error()))
 		return utilerrors.NewAggregate(errs)
```

Comment:
Minor nit as you're only changing the error message and the implementation in this PR, but this seems strange to define a list of errors, run the Exec method and if the error isn't nil, append to an empty list and return that list. Do we want to instead just return an error here using the formatted error message you currently have and define the `errs` variable further down?